pkg/jerryscript/Makefile.include

```diff
@@ -1,2 +1,8 @@
 INCLUDES += -I$(PKGDIRBASE)/jerryscript/jerry-core/include
 INCLUDES += -I$(PKGDIRBASE)/jerryscript/jerry-ext/include
+
+ifneq (,$(filter cortex-m%,$(CPU_ARCH)))
+  # There is a linking issue to `_setjmp()` and `stderr` when compiling
+  # for Cortex-M with LLVM/clang
```

Comment:
OK I see the error with `samr21-xpro`:

    make -C examples/javascript BOARD=samr21-xpro distclean all TOOLCHAIN=llvm

Reply:
Mhh... now that I went through all that trouble fixing LLVM in jerryscript, I kind of want to know what's causing this, especially since [their test](https://github.com/jerryscript-project/jerryscript/pull/2477) with RIOT doesn't seem to have that problem :-/.

Reply:
Posted in the wrong PR before:

With #9821 it looks like I can correctly compile examples/javascript with llvm both on ubuntu 16.04 and in the docker image with BOARD=samr21-xpro.